packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2131,8 +2129,6 @@ class ClipboardStatusNotifier extends ValueNotifier<ClipboardStatus> with Widget
   }) : super(value);
 
   bool _disposed = false;
-  /// True if this instance has been disposed.
-  bool get disposed => _disposed;
```

Comment:
This is used by the listeners to decide whether they should call removeListener when they are disposed

This is no longer needed because removeListener is allowed to be called after it is disposed now